### ISSUE TITLE
fine tuning yoloWordv2-yolov8s && val yoloWord in LVIS

### ISSUE THREAD

**Issue by mufeng8023** [ultralytics/ultralytics#19928]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hi, when I fine tuning yoloWordv2-s, the evaluation result on LVIS was very small, the overall mAP50 was about 0.0946; Using the official weights alone to evaluate LVIS is the same result, but the evaluation of COCO is correct, how should the correct configuration can get the correct result on LVIS?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @mufeng8023, thank you for your interest in Ultralytics 🚀! We recommend starting with the [Docs](https://docs.ultralytics.com/) for guidance, including tips on [Python usage](https://docs.ultralytics.com/usage/python/) and [CLI usage](https://docs.ultralytics.com/usage/cli/). These resources may already have answers to your question.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us investigate the issue.

If this is a ❓ Question about fine-tuning or evaluation, please share additional details so we can better assist you:
- Specific steps or configurations you used for fine-tuning
- Evaluation command and configuration
- Relevant logs or outputs
- Dataset examples if possible

For fine-tuning, ensure you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/), which includes guidance on hyperparameters, dataset preparation, and evaluation settings.

Additionally, join the Ultralytics community to connect with other users! For real-time conversations, visit [Discord](https://discord.com/invite/ultralytics) 🎧. For detailed discussions, check out [Discourse](https://community.ultralytics.com/). Or, share ideas and experiences on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

Please upgrade to the latest `ultralytics` package to ensure you’re using the most up-to-date features and fixes. Use the following command in a [**Python>=3.8**](https://www.python.org/) environment:

```bash
pip install -U ultralytics
```

This will also ensure compatibility with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Verify your issue persists with the latest version.

## Environments

YOLO can be run in any of the following verified environments, complete with necessary dependencies:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. These tests verify the correct operation of YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response to guide you while an Ultralytics engineer reviews your issue and provides further assistance. Let us know if you have more details to share in the meantime! 😊

**Comment by mufeng8023** [comment]
As a result of my experiment, the following three options must be specified when verifying yolowrd: data="lvis.yaml", split='minival', save_json=True. The above options are evaluated on minival. And the data set needs to have annotations directory and corresponding lvis_v1_minival.json and lvis_v1_val.json, evaluation results see the result of using json evaluation. The results of my evaluation using this method are basically consistent with those in the paper;

![Image](https://github.com/user-attachments/assets/6e2c521e-20d2-47ba-b755-29d69a57c9a2)

**Comment by glenn-jocher** [comment]
@mufeng8023 thank you for sharing your findings! You're correct that evaluating YOLO-World on LVIS requires specific configuration: using the `split='minival'` argument and ensuring proper dataset annotations. This aligns with our LVIS dataset documentation which details the minival split requirements. For others facing similar issues, please ensure your validation command includes:

```python
model.val(data='lvis.yaml', split='minival', save_json=True)
```

And verify your dataset structure matches the LVIS YAML specifications. The Ultralytics team appreciates community contributions like this that help improve everyone's experience! 🚀

**Comment by mufeng8023** [comment]
By the way, here's the code where I got the correct rating,
![Image](https://github.com/user-attachments/assets/61c45ffc-6509-4521-b3ef-598f62d5baf4)

**Comment by mufeng8023** [comment]
By the way, here's the code where I got the correct rating,

![Image](https://github.com/user-attachments/assets/c0a18c0f-48d4-47ba-8d8c-1df769381528)

**Comment by glenn-jocher** [comment]
Thank you for sharing your solution! Your approach aligns with our LVIS validation recommendations in the [LVIS dataset documentation](https://docs.ultralytics.com/datasets/detect/lvis/) which specifies using `split='minival'` and proper annotation paths. The configuration you've shown matches our validation parameters documentation in [YOLO configuration settings](https://docs.ultralytics.com/usage/cfg/#validation-settings). This will help others implement LVIS evaluation correctly - we appreciate you contributing these insights to the community! 🚀
